content/blog/2023-03-23-when-a-date-breaks-booting-the-kernel.md

```diff
@@ -0,0 +1,213 @@
+Title: When a date breaks booting the kernel
+Date: 2023-03-23 00:00:00
+Authors: Benjamin Gray
+Category: Development
+Tags: linux
+
+## The setup
+
+I've recently been working on internal CI infrastructure for testing kernels before sending them to the mailing list. As part of this effort, I became interested in reproducible builds. Minimising the changing parts outside of the source tree itself could improve consistency and ccache hits, which is great for trying to make the CI faster and more reproducible.
```

Comment:
> 'Minimising the changing parts'

If in one sentence you can describe/motivate what moving parts exist that could affect ccache hits etc this may help